src/clj/rems/application/commands.clj

```diff
@@ -74,6 +74,9 @@
   (assoc CommandBase
          :member {:name s/Str
                   :email s/Str}))
+(s/defschema PromoteToApplicantCommand
```

Comment:
Yeah I wanted to name the command & event differently because I thought the event might get generated from other sources as well. Also the command is more limited in scope than the event. I'd be fine with `ChangeApplicant` or `MemberToApplicant`.

Reply:
- Promote sounds to me that there can be several applicants after it. 
- Change applicant sounds like the old is not anymore (matches current implementation).
- Member to applicant could support multiple applicants as well.

Maybe change is the best match of these three ideas?

Reply:
MakeApplicant?

Reply:
Renamed to ChangeApplicant!